presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -187,3 +187,46 @@ Accessors
 .. function:: ST_NumInteriorRing(Geometry) -> bigint
 
     Returns the cardinality of the collection of interior rings of a polygon.
+
+=====================
+Bing Tiles
+=====================
+
+A set of functions to convert between geometries and
+ `Bing tiles <https://msdn.microsoft.com/en-us/library/bb259689.aspx>`_.
+
+.. function:: bing_tile(x, y, zoom_level) => BingTile
+
+    Given XY coordinates and a zoom level, returns the corresponding Bing tile.
+    Zoom levels from 1 to 23 are supported.
+
+.. function:: bing_tile(quadKey) => BingTile
+
+    Given a QuadKey, returns the corresponding Bing tile.
+
+.. function:: bing_tile_coordinates(tile) => row(x, y)
+
+    Given a Bing tile, returns XY coordinates of that tile.
+
+.. function:: bing_tile_polygon(tile) => Geometry
+
+    Given a Bing tile, returns the polygon representation of that tile.
+
+.. function:: bing_tile_quadkey(tile) => varchar
+
+    Given a Bing tile, returns its QuadKey.
+
+.. function:: bing_tile_zoom_level(tile) => bigint
```

Comment:
zoom level is <=23, so why doesn't this function return `integer`? (For now, I know that we use Java `long` to represent the `integer` type, but going forward that may change.)

Reply:
We should make this return tinyint